docs/_templates/class.rst

```diff
@@ -9,9 +9,15 @@
 .. |bbox_labels| replace:: ``bbox_labels`` (:obj:`numpy.array`): a numpy array (N) strings,
    representing the labels of detected objects. The order corresponds to `bboxes` and `bbox_scores`.
 
-.. |bbox_scores| replace:: ``bbox_scores`` (:obj:`numpy.array`): a numpy array (N). [0,1]
+.. |bbox_scores| replace:: ``bbox_scores`` (:obj:`numpy.array`): a numpy array (N) [0,1]
    of the confidence scores for detected objects. The order corresponds to `bboxes` and `bbox_labels`.
 
+.. |btm_midpoint| replace:: ``btm_midpoint`` (:obj:`numpy.array`): a numpy array (N)
+   of a single point of reference of bounding boxes for use in zone analytics
```

Comment:
can check this again? I think its a list of tuples (x,y) instead of a numpy array (N)

Reply:
you're right, updated accordingly